ivy/functional/frontends/tensorflow/math.py

```diff
@@ -657,3 +657,12 @@ def zeta(x, q, name=None):
 def greater_equal(x, y, name=None):
     x, y = check_tensorflow_casting(x, y)
     return ivy.greater_equal(x, y)
+
+
+@to_ivy_arrays_and_back
+def in_top_k(targets, pred, k):
+    top_k = ivy.top_k(targets, k)
+    Ayad = []
+    for val in targets:
+        Ayad.extend([val in top_k.values])
+    return ivy.array(Ayad)
```

Comment:
```suggestion
    return ivy.any(top_k.T == targets, axis=0)

```
We can move away from the for loop because it is inefficient and use `ivy.any(...)` instead, try tweaking the suggestion to do the job, either by triggering the `.T` or changing the `axis` values.

Also, it is better to have more conventional names for variables to convey what they represent, `Ayad` can be replaced with `ret`